akka-http/src/main/scala/com/avast/grpc/jsonbridge/akkahttp/AkkaHttp.scala

```diff
@@ -41,38 +42,87 @@ object AkkaHttp extends SprayJsonSupport with DefaultJsonProtocol {
       .map(_ / Segment / Segment)
       .getOrElse(Segment / Segment)
 
+    logger.info(s"Creating Akka HTTP service proxying gRPC services: ${bridge.servicesNames.mkString("[", ", ", "]")}")
+
     post {
       path(pathPattern) { (serviceName, methodName) =>
-        extractRequest { req =>
-          req.header[`Content-Type`] match {
+        extractRequest { request =>
+          val headers = request.headers
+          request.header[`Content-Type`] match {
             case Some(`JsonContentType`) =>
-              entity(as[String]) { json =>
-                val methodCall = bridge
-                  .invoke(GrpcMethodName(serviceName, methodName), json, mapHeaders(req.headers))
-                  .toIO
-                  .unsafeToFuture()
+              entity(as[String]) { body =>
+                val methodName0 = GrpcMethodName(serviceName, methodName)
```

Comment:
`0` suffix of variable name here and like below - probably result of an automatic refactoring.

Reply:
that's intentional -- I can't reuse (shadow) the variable `methodName`
open to suggestions how to improve this :)

Reply:
E.g. renamed `methodName` to `methodNameAsString` or something like this - it's IMHO better than `0` suffix :)